Emmanuelaaron/math-magicians
# calculator UI

## Sets Up the Calculator UI

I have set up the Calculator UI as per the design provided using a class-based component

### Tools used
- React
- Eslint
- Stylint 

### To test this branch Locally
- Clone this repo ```git clone git@github.com:Emmanuelaaron/math-magicians.git```
- Checkout to ths branch ```git checkout feature-2"
- Install all the dependencies ```npm install``
- Run the server ```npm start```
- The browser should be open and the UI of a calculator displayed
